config/routes.rb

```diff
@@ -4,11 +4,13 @@
   ActiveAdmin.routes(self)
   devise_for :users
   root "movies#index"
-  resources :movies, only: [:index]
+  resources :movies, only: [:index] do
+    resources :watched_buttons, only: [:create, :destroy]
```

Comment:
ここが動作しない原因ですね。`resources` ではなく `resource` を使用しましょう。

```suggestion
    resource :watched_buttons, only: [:create, :destroy]
```